botocore/handlers.py

```diff
@@ -975,11 +976,11 @@ def inject_api_version_header_if_needed(model, params, **kwargs):
 
     ('before-call.s3', add_expect_header),
     ('before-call.glacier', add_glacier_version),
-    ('before-call.apigateway', add_accept_header),
+    ('before-call.api-gateway', add_accept_header),
```

Comment:
I'm assuming we don't have to apply this to api gateway v2 as well? I know for a while those two shared the same endpoint prefix so I don't know if we had an alias that would trigger the handler for api gateway v2 as well.

Reply:
I checked but it doesn't seem like this event is triggering for API Gateway V2 on `v2` or the `develop` branch. Not sure if it ever has or should be hooked up to APIG V2.